doc/performance_counters.rst

```diff
@@ -3,144 +3,48 @@
+The remaining counters are disabled.
+
++------------------+----------------+---------------------------------------------------------+
+| Counter          | CSR Address    | Event Description                                       |
++==================+================+=========================================================+
+| mcycle(h)        | 0xB00 (0xB80)  | Number of cycles                                        |
++------------------+----------------+---------------------------------------------------------+
+| minstret(h)      | 0xB02 (0xB82)  | Number of instructions retired                          |
++------------------+----------------+---------------------------------------------------------+
+| mhpmcounter3(h)  | 0xB03 (0xB83)  | Number of cycles waiting for data memory                |
++------------------+----------------+---------------------------------------------------------+
+| mhpmcounter4(h)  | 0xB04 (0xB84)  | Cycles waiting for instruction fetches, i.e., number of |
+|                  |                | instructions wasted due to non-ideal caching            |
++------------------+----------------+---------------------------------------------------------+
+| mhpmcounter5(h)  | 0xB05 (0xB85)  | Number of data memory loads. Misaligned accesses are    |
+|                  |                | counted twice                                           |
++------------------+----------------+---------------------------------------------------------+
+| mhpmcounter6(h)  | 0xB06 (0xB86)  | Number of data memory stores. Misaligned accesses are   |
+|                  |                | counted twice                                           |
++------------------+----------------+---------------------------------------------------------+
+| mhpmcounter7(h)  | 0xB07 (0xB87)  | Number of unconditional jumps (j, jal, jr, jalr)        |
++------------------+----------------+---------------------------------------------------------+
+| mhpmcounter8(h)  | 0xB08 (0xB88)  | Number of branches (conditional)                        |
++------------------+----------------+---------------------------------------------------------+
+| mhpmcounter9(h)  | 0xB09 (0xB89)  | Number of taken branches (conditional)                  |
++------------------+----------------+---------------------------------------------------------+
+| mhpmcounter10(h) | 0xB0A (0xB8A)  | Number of compressed instructions retiredz              |
++------------------+----------------+---------------------------------------------------------+
+
+By default, all these counters are enabled after reset.
+They can be individually enabled or disabled by overwriting the corresponding bit in the ``mcountinhibit`` CSR at address ``0x320`` with 0 or 1, respectively.
+To enable/disable ``mcycle(h)``, bit 0 must be written. For ``minstret(h)``, it is bit 2. For event counter ``mhpmcounterX(h)``, it is bit X.
```

Comment:
Could you add a reference to the RISC-V Privileged Spec here and mention that we implement exactly that?

Reply:
Done